commands/scp.go

```diff
@@ -0,0 +1,136 @@
+package commands
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/codegangsta/cli"
+	"github.com/docker/machine/libmachine"
+	"github.com/docker/machine/log"
+)
+
+var (
+	ErrMalformedInput = fmt.Errorf("The input was malformed")
+)
+
+var (
+	// TODO: possibly move this to ssh package
+	baseSSHOptions = []string{
+		"-o", "IdentitiesOnly=yes",
+		"-o", "StrictHostKeyChecking=no",
+		"-o", "UserKnownHostsFile=/dev/null",
+		"-o", "LogLevel=quiet", // suppress "Warning: Permanently added '[localhost]:2022' (ECDSA) to the list of known hosts."
+	}
+)
+
+func getInfoForScpArg(hostAndPath string, mcn libmachine.Machine) (*libmachine.Host, string, []string, error) {
+	// TODO: What to do about colon in filepath?
+	splitInfo := strings.Split(hostAndPath, ":")
```

Comment:
Actually, I'm going to punt on this for now.  I think it will be OK since e.g. `$PATH` uses `:` as a delimiter, so I'm pretty sure it's not very common to have in UNIX filepaths, which are pretty much assumed to be what we're handling here at this point..